src/components/common/infocolourBar/infoColourBar.scss

```diff
@@ -20,7 +20,13 @@
         background-color: var(--R100);
         border: 1px solid var(--R200);
     }
+    .info-colour-bar svg{
+        
+        height: 20px;
+        width: 27px;
```

Comment:
This should not be the right way to do

Reply:
This is the same Issue we had a discussion on regarding the icon, getting smaller icon, so sole suggested to target it individually.

Reply:
Is width right here? Usually, we go for same height width for icons.

Reply:
Also, overrides should not be common class. It should be in css/scss of the component wherever it's used